service/src/main/java/bio/terra/workspace/amalgam/tps/TpsApiDispatch.java

```diff
@@ -18,12 +19,9 @@
 import bio.terra.workspace.generated.model.ApiTpsPolicyInput;
 import bio.terra.workspace.generated.model.ApiTpsPolicyInputs;
 import bio.terra.workspace.generated.model.ApiTpsPolicyPair;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
-import java.util.UUID;
-import java.util.stream.Collectors;
+import com.google.common.collect.ArrayListMultimap;
+import com.google.common.collect.Multimap;
+import java.util.*;
```

Comment:
nit: avoid wildcard imports

Reply:
FYI - If you search Terra onboarding doc for "Java imports", it tells you what Intellij setting to change

Reply:
thanks, that was annoying.